### DISCUSSION TITLE
Welcome to Karttapullautin Discussions!

### DISCUSSION THREAD

**Discussion by rphlo** [karttapullautin/karttapullautin#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stefankinell** [comment]
Stefan here, heavy Kartapullautin user. Mainly as a tool to render the automatic generated map of Sweden at https://kartor.gokartor.se. Have written some guides in Swedish as well on how to use kartapullautin e.g. https://stefansolsida.wordpress.com/2017/06/10/att-jobba-med-karttapullautin-tips-och-trix-for-batchkorningar-och-vilka-fel-man-vill-undvika/ and https://stefansolsida.wordpress.com/2020/06/22/2-laserdata-skapa-kurvor/ 

Have not coded that much in a while, tinkering a bit with some scripts from time to time, so I think my contributions are more on the user side, and ideas of what one would like to be able to do.

Right now struggling with geting it working on my Mac, can't get through making the libgd modules work. Hope to come past that soon so I can test and eventually contribute as well.

**Reply by jaluzdi** [reply]
My name is Juantxo. Born in Bilbao (north of Spain) and living in a nearby village. 
I am one of the collaborators (just putting my PC to process tiles) in [https://mapant.es](https://mapant.es)
Manu, the creator of mapant Spain, told us about this rusty-pullauta, and the improvements in processings, and I wanted to give it a try.
I am really impressed with the speed processing improvements so far.
I am a Linux and Windows user, and professionally a COBOL/SQL programmer. 
I have little experience with bash sccripting in Linux and with *.bat (cmd) and *.ps1 (powershell) in Windows.

P.D. Thanks to @stefankinell for his warm welcome in one of the issues closed today. And many thanks to @rphlo for this great job with rusty-pullauta
